src/project/internal/projectactionscontroller.cpp

```diff
@@ -1203,7 +1203,7 @@ bool ProjectActionsController::askIfUserAgreesToOpenProjectWithIncompatibleVersi
 
 bool ProjectActionsController::askIfUserAgreesToOpenCorruptedProject(const String& projectName, const std::string& errorText)
 {
-    std::string title = mtrc("project", "File “%1” is corrupted").arg(projectName).toStdString();
+    std::string title = mtrc("project", "File "%1" is corrupted").arg(projectName).toStdString();
```

Comment:
Needs to be
`std::string title = mtrc("project", "File \"%1\" is corrupted").arg(projectName).toStdString();`

Reply:
Done.